### PR TITLE
update sig-storage owners

### PR DESCRIPTION
**Pull request by msau42**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
add sig-storage reviewers, remove inactive sig-storage reviewers, remove redundant owners files

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -135,29 +135,30 @@ aliases:
...
     - gnufied
-    - verult
     - humblec
-    - xing-yang
-    - chrishenzie
-    - Jiawei0227
+    - jsafrane
+    - jingxu97
     - mattcary
+	- mauriciopoppe
```
Comment (aojea) [inline OWNERS_ALIASES:153]: wrong indention?

Comment (msau42) [reply OWNERS_ALIASES]: thanks fixed

**Comment by aojea** [comment]
this went stale :/

**Pull request merged**